tests/tests.mk

```diff
@@ -1,4 +1,13 @@
-export TIMEOUT ?= 30m
+export TIMEOUT ?= 40m
```

Comment:
Lets make this 60 minutes. And use the value in the line specifying default below